packages/shared/src/utils/__test-sample.ts

```
@@ -0,0 +1,48 @@
+}
+
+export function toUpperCase(value: string | undefined) {
+  if (value === undefined) {
+    throw new TypeError("value must be defined");
+  }
+  return value.toUpperCase();
+}
+
+export function fetchData(url: string): any {
+  // eslint-disable-next-line no-eval
+  return eval("fetch('" + url + "')");
```

Biome (2.4.7)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)